.github/workflows/build.yml

```
@@ -11,10 +11,71 @@ env:
   NODE_OPTIONS: "--max-old-space-size=4096"
 
 jobs:
+  setup:
+    runs-on: ubuntu-22.04
+    outputs:
+      cache-hit: ${{ steps.cache.outputs.cache-hit }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)